perfkitbenchmarker/packages/ant.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2014 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""Module containing Ant installation and cleanup functions."""
+
+from perfkitbenchmarker import vm_util
+
+ANT_TAR_URL = ('www.pirbot.com/mirrors/apache//ant/binaries/'
+               'apache-ant-1.9.6-bin.tar.gz')
+
+
+def _Install(vm):
+    """Installs the Ant package on the VM."""
+    vm.Install('wget')
+    vm.RemoteCommand('cd {0} && '
+                     'wget {1} && '
+                     'sudo tar -C /opt/ -zxf apache-ant-1.9.6-bin.tar.gz && '
```

Comment:
Would it be possible to install `ant` in `vm_util.VM_TMP_DIR`? It makes the cleanup a bit easier.